src/sgame/sg_admin.cpp

```diff
@@ -5774,11 +5774,12 @@ void G_admin_cleanup()
 static void BotUsage( gentity_t *ent )
 {
 	static const char bot_usage[] = QQ( N_( "^3bot:^* usage: bot add (<name> | *) (aliens | humans) [<skill level> [<behavior>]]\n"
-	                                        "            bot fill <count> [<team> [<skill level>]]\n"
+	                                        "            bot behavior (<name> | <slot#>) <behavior>\n"
```

Comment:
The idea is to put the most used commands at the beginning rather than alphabetize